ansible/roles/llvm10/tasks/main.yml

```diff
@@ -30,6 +30,7 @@
     name:
       - clang-10
       - clangd-10
+      - clang-tidy-10
```

Comment:
`clang-tidy-10` does not ship with `clang-tools-10` and it has to be installed separately. For the tests to pass then, we need to rebuild `cpp-root`, `base`, and `cli` containers.

Reply:
Is it really part of the base requirements for Faasm, or just the CLI container? I would say it's really only used in the CLI container, so should live in the Dockerfile for that. This file is too overblown as it is (e.g. we don't need `clang-tools` in the `worker` container)